manager/main.go

```diff
@@ -187,7 +187,7 @@ func run(namespace string, metricsAddr string, enableLeaderElection bool,
 			return 1
 		}
 		defer watcher.Close()
-		// watch /tmp/adminconfig directory for changes
+		// watch /data/adminconfig directory for changes
```

Comment:
$DATA/adminconfig ? or GetDataDir() + "/adminconfig"

Reply:
I changed that.